### ISSUE TITLE
[Pathway Issue]: WP5441 - connection between receptor tyrosine kinase and PI3K

### ISSUE THREAD

**Issue by aish181095** [wikipathways/wikipathways-help#104]
### What is the issue with the pathway?

In this pathway the steps in between these two nodes are not clear. The link between receptor tyrosine kinase and PI3K is well established with the signalling steps involved. Do we want to include those here? Although they are not specified on the pathway diagram it was derived from.

### What kind of content issue is this?

- [X] Missing content
- [ ] Incorrect content
- [ ] Duplicated content
- [ ] Other

### Type of revision

- [ ] Major
- [X] Minor
- [ ] Deletion

**Comment by khanspers** [comment]
I agree that the signaling steps from RTK and PI3K should be included. Additional improvements:
- Include all AKT protein family members in the actual diagram, as opposed to in the legend on the side.
- Describe the meaning of the green triangle in a legend, and maybe also in the description.

**Comment by eweitz** [comment]
Done!  Thanks for the notes.  Below I discuss tradeoffs of a style suggestion, and show views before and after this change.  

### Extracted or inlined gene families
> Include all AKT protein family members in the actual diagram, as opposed to in the legend on the side.

I've done that as shown below.  In many cases, though, I think extracting gene family members to a legend would give the diagrams more explanatory power than inlining family members in the diagram.

Consider the "receptor tyrosine kinase" gene family in this diagram.  [Wikipedia notes](https://en.wikipedia.org/wiki/Receptor_tyrosine_kinase#Classes) roughly "20 different RTK classes have been identified".  The literature Wikipedia cites -- Table 2 in [PMC4620971](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4620971/) -- lists 46 genes in that family.  PI3K is a family of 14 genes.  AKT is a family of 3 genes.  Inlining those 3 gene-family nodes to 63 gene nodes in the pathway itself seems like it would dilute the explanatory power of the visualization.

Extracting the genes in each family to the legend would often have benefits:

* **Faster understanding**.  The source diagram in this case is 10 simple nodes.  That's quicker and easier to visually comprehend than 7 simple nodes and 3 group nodes that each have 3-46 member nodes.

* **Literature fidelity**.  Inlining family members would often make WikiPathways diagrams look and feel substantially different than their source diagrams, which usually refer to gene families instead of their full list of members.

* **Easier authoring**.  This diagram was sourced [from PFOCR](https://pfocr.wikipathways.org/figures/PMC8684509__OMCL2021-9911537.012.html), which is a fabulous catalyst for refining raw scientific figures into computable pathways.  Simply putting groups into the legend -- i.e. not needing to think about how inlined families affect local and global pathway layout -- makes it easier to scale the throughput of converting figures into pathways.

* **Layout flexibility**.  Legends can be hidden in GPML clients without affecting the main pathway graphics.  When families are inlined into the pathway graphics, such powerful flexibility for pathway graphic becomes infeasible; inlined groups can't be hidden without affecting the pathway graphics.  Extracting family members into the legend means pathways would much less often require zooming in downstream viewports, and expand the reusability of WikiPathways graphics.

I don't feel strongly about this -- legends have a lot of variety in WikiPathways!  I hope the above is helpful food for thought.

### Before
https://classic.wikipathways.org/index.php?title=Pathway:WP5441&oldid=128352
<img width="678" alt="WP5441_before__WikiPathways_2024-02-05" src="https://github.com/wikipathways/wikipathways-help/assets/1334561/659093e8-2ebc-4b47-8c88-27c5ff0f2ca2">

### After
https://classic.wikipathways.org/index.php?title=Pathway:WP5441&oldid=128482
<img width="568" alt="WP5441_after__WikiPathways_2024-02-05" src="https://github.com/wikipathways/wikipathways-help/assets/1334561/60f50df9-26df-4204-aa68-0d97ce4740dd">

**Comment by khanspers** [comment]
Great discussion points @eweitz! I appreciate the detailed descriptions and have some comments as well:

While representing protein families in a legend does make the pathway **easier to author** and (potentially) **easier to understand**, it does have some other drawbacks. The most obvious issue is that pathways with unconnected nodes in a legend don't work well in downstream network analysis tools, since the nodes are not part of the connected network.

In terms of **literature fidelity**, I would caution that many published pathway figures can be problematic; there are no standards for how they are drawn, they often contain mistakes, or are just poorly drawn. While it might make sense for some use cases to try and match the pathway model to the figure, I would say that in general this should _not_ be a goal for pathway curators. Rather the pathway model should try to capture the biology accurately. You can see some examples of pathway figures that have been translated to WikiPathways models [here](https://pfocr.wikipathways.org/browse/wikipathways.html) (click on the WikiPathways link under the image); most of them look quite different than the original figure. (Side note: I'm working on adding your two latest pathways to this list....)

These points are not meant as counter points necessarily, just to bring another perspective to this discussion. In the end, WikiPathways does not impose a specific style; it's up to each author how to represent protein families, how to handle pathway layout tradeoffs, or what to put in legends etc.
